public/app/features/query/components/QueryEditorRow.tsx

```diff
@@ -45,6 +46,7 @@ interface Props<TQuery extends DataQuery> {
   onRunQuery: () => void;
   visualization?: ReactNode;
   hideDisableQuery?: boolean;
+  draggable?: boolean;
```

Comment:
We added this so the component can be tested in isolation. If draggable is true, this component things it should be part of a draggable context and rendering fails.